modules/ipi-install-removing-the-provisioner-node.adoc

```diff
@@ -0,0 +1,13 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc
+
+[id="ipi-install-removing-the-provisioner-node_{context}"]
+= Removing the provisioner node
+
+Optionally, the provisioner node can be removed after installation.
+Make sure to back up the following files:
```

Comment:
The three listed items are not all files - 1 file, 1 directory, 1 pair - so maybe best just go with 
'Make sure to back up the following:'

Reply:
Updated as suggested.